Tasks/Common/kubernetes-common-v2/utility.ts

```diff
@@ -1,9 +1,57 @@
 import * as os from 'os';
+import * as tl from 'azure-pipelines-task-lib/task';
+import { IExecSyncResult } from 'azure-pipelines-task-lib/toolrunner';
 
 export function getExecutableExtension(): string {
     if (os.type().match(/^Win/)) {
         return '.exe';
     }
 
     return '';
+}
+
+export function isEqual(str1: string, str2: string, ignoreCase?: boolean): boolean {
+    if (str1 == null && str2 == null) {
+        return true;
+    }
+
+    if (str1 == null) {
+        return false;
+    }
+
+    if (str2 == null) {
+        return false;
+    }
```

Comment:
Convert this to `if (!str1 || !str2) return false`

Reply:
Done.